node/silkworm/db/access_layer.cpp

```diff
@@ -502,22 +578,62 @@ std::optional<ChainConfig> read_chain_config(mdbx::txn& txn) {
     return ChainConfig::from_json(json);
 }
 
+static Bytes head_header_key() {
+    std::string table_name = db::table::kHeadHeader.name;
+    Bytes key{table_name.begin(), table_name.end()};
+    return key;
+}
+
+void write_head_header_hash(mdbx::txn& txn, const evmc::bytes32& hash) {
+    write_head_header_hash(txn, hash.bytes);
+}
+
 void write_head_header_hash(mdbx::txn& txn, const uint8_t (&hash)[kHashLength]) {
     Cursor target(txn, table::kHeadHeader);
-    mdbx::slice key(db::table::kLastHeaderKey);
-    target.upsert(key, to_slice(hash));
+    Bytes key = head_header_key();
+    auto skey = db::to_slice(key);
+
+    target.upsert(skey, to_slice(hash));
 }
 
 std::optional<evmc::bytes32> read_head_header_hash(mdbx::txn& txn) {
     Cursor src(txn, table::kHeadHeader);
-    mdbx::slice key(db::table::kLastHeaderKey);
-    auto data{src.find(key, /*throw_notfound=*/false)};
+    Bytes key = head_header_key();
+    auto skey = db::to_slice(key);
+    auto data{src.find(skey, /*throw_notfound=*/false)};
     if (!data || data.value.length() != sizeof(evmc::bytes32)) {
```

Comment:
Probably instead of `sizeof(evmc::bytes32)` is better to use the defined constant `kHashLength`

Reply:
ok